Add bob.jar plugins

AGulev opened the pull request:
I added it to alpha for now. Usually we need to catch it on early stage, later stages should be fine if alpha is fine

JCash commented:
"I added it to alpha for now. Usually we need to catch it on early stage, later stages should be fine if alpha is fine"

Same goes for the rest of the tests as well.

Note that this tests _both_ the sdk, and the server.
I don't think we should keep certain tests to just one configuration.

AGulev commented:
> "I added it to alpha for now. Usually we need to catch it on early stage, later stages should be fine if alpha is fine"
> 
> Same goes for the rest of the tests as well.
> 
> Note that this tests _both_ the sdk, and the server. I don't think we should keep certain tests to just one configuration.

it makes sense, I added it to the other configs
